### PR TITLE
GEN-1447 | Confirm price intent + add to cart in Widget

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

- After completing price calculator; confirm price intent + add default offer to cart

- Then navigate to the sign page in the flow

## Justify why they are needed

- Progress!

- I skipped all the tracking for now -- let's discuss how we want to handle that

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3544** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3544?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #3545** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3545?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #3546** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3546?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #3547** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3547?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #3548** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3548?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3548?utm_source=stack-comment).

File: apps/store/src/utils/PageLink.ts [hunk 1]
```diff
@@ -285,6 +286,12 @@ export const PageLink = {
       ORIGIN_URL,
     )
   },
+  widgetSign: (params: WidgetSignParams) => {
```
Comment (guilhermespopolin) [inline apps/store/src/utils/PageLink.ts:289]: I should probably rename the page I've been working on to `sign` instead of checkout page.

**Review by guilhermespopolin — APPROVED** [review]
Well done

File: apps/store/src/utils/PageLink.ts (hunk 1 above)
Comment (robinandeer) [reply apps/store/src/utils/PageLink.ts:289]: Oh yeah let's agree on a term -- I just picked it from Figma without thinking too much about it

**Review by gustaveen — APPROVED** [review]
LGTM! 🙌

**Comment by robinandeer** [comment]
### Merge activity

* **Nov 14, 3:38 PM**: @robinandeer merged this pull request with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3548).

**Pull request merged**